modules/swagger-codegen/src/main/resources/swift/Models.mustache

```diff
@@ -145,6 +145,11 @@ class Decoders {
         }
         // Decoder for {{{classname}}}
         Decoders.addDecoder(clazz: {{{classname}}}.self) { (source: AnyObject) -> {{{classname}}} in
+{{#isArrayModel}}
+            let sourceArray = source as! [AnyObject]
+            return sourceArray.map({ Decoders.decode(clazz: {{{arrayModelType}}}.self, source: $0) })
```

Comment:
I'm not sure this will work for a scenario where the array is composed of enum objects as those don't have decoders. You can see my initial workaround for this here:

https://github.com/hexelon/swagger-codegen/blob/swift-3/modules/swagger-codegen/src/main/resources/swift/Models.mustache#L163